rapidpro-org/org.proto

```diff
@@ -54,4 +56,4 @@ message OrgUpdateRequest {
     optional bool is_multi_user = 10;
     optional bool is_multi_org = 11;
     optional bool is_suspended = 12;
-}
+}
```

Comment:
missing an endline blank here

Reply:
Sorry, I didn't know that in the proto I also needed this line.